tictactoe.js

```diff
@@ -0,0 +1,131 @@
+const Player = name =>{
+    return {name}
+}
+
+let Player1 = Player('Jeff');
+let Player2 = Player('Rama')
+const squares = document.querySelectorAll('.squares')
+const form = document.querySelector('.form')
+const replay = document.querySelector('.replay')
+let name1 = document.querySelector('.player1')
+let name2 = document.querySelector('.player2')
+let counter = 0
+let result = document.querySelector('.result')
+
+const game = (() => {
+  let gameboard = [];
+
+  const check = () => {
+    if(gameboard[0] == gameboard[1] && gameboard[0]== gameboard[2]) {
+      return gameboard[0]
+      } 
+      if(gameboard[1] == gameboard[7] && gameboard[4]== gameboard[1]) {
+        return gameboard[1]
+        } 
+        if(gameboard[2] == gameboard[5] && gameboard[2]== gameboard[8]) {
+          return gameboard[8]
+          } 
+          if(gameboard[0] == gameboard[3] && gameboard[3]== gameboard[6]) {
+            return gameboard[3]
+            } 
+      return false
+  }
+
+  const check2 = () => {
+    if(gameboard[3] == gameboard[4] && gameboard[3]== gameboard[5]) {
+      return gameboard[3]
+      } 
+      if(gameboard[6] == gameboard[7] && gameboard[6]== gameboard[8]) {
+        return gameboard[6]
+        } 
+        if(gameboard[0] == gameboard[4] && gameboard[0]== gameboard[8]) {
+          return gameboard[0]
+          } 
+          if(gameboard[2] == gameboard[4] && gameboard[2]== gameboard[6]) {
+            return gameboard[2]
+            } 
+      return false
+  }
+
+  
+  const clearboard = () => {
+    gameboard = []
+  }
+  
+  const play = (tool, position) => {
+     gameboard[position] = tool
+  };
+  return {
+   check,check2, clearboard, play
+  };
+})();
+
+const display = (() => {
+  const endGame = () => {
+    squares.forEach(button => {
+      button.disabled = true
+    })
+  }
+  const startNew = () => {
+    endGame();
+    form.style.display = 'block'
+    replay.style.display = 'none'
+      }
+      const start = () => {
+
+        if(name1.value != '' && name2.value != '') {
+          Player1 = Player(name1.value)
+          Player2 = Player(name2.value)
+          return true
+        }
+        return false
+      }
+  return {
+    endGame, start, startNew
+  }
+})();
+
+const Umpire = (() => {
+  const startGame = () => {
+    form.style.display = 'none'
+    squares.forEach(square => {
+      square.addEventListener('click', () => {
+        if (square.textContent == '') {
+          document.querySelector('.warn').style.display = 'none'
+          counter % 2 == 0? square.textContent = 'X' : square.textContent = 'O'
+          counter += 1
+          game.play(square.textContent , parseInt(square.id - '1',10))
+        Umpire.sayResult() || counter == 9 ? display.endGame() : null
+        }
+    })
+    })    
+  }
+  const sayResult = () => {
+    result.innerHTML =''
+    replay.style.display = 'block'

```

Comment:
Using [`classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) we could add and remove class, doing this help keep styling logic in the CSS files.

> Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)